src/styles/main.scss

```diff
@@ -1 +1,12 @@
-// put code here
+$color-base: #a5978a;
+$color-hour: #075773;
+$color-minute: #188193;
+$color-dots: #555653;
+
+@import "./components/watch";
+
+.page {
+  &__watch {
+    margin: auto;
+  }
```

Comment:
`margin: auto` works correctly only if parent has `display: flex`. Here it acts like `margin: 0 auto`